packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -271,7 +271,18 @@ beforeAll( async () => {
 } );
 
 afterEach( async () => {
-	await runAxeTestsForBlockEditor();
+	/**
+	 * TODO: In the listed test, axe core will often (but not always, and never
+	 * locally) fail with the error "failed to inject axe-core into the frame
+	 * with source about:blank". We've tried several things to fix this issue,
+	 * but it's been a big pain so we need to skip it for now.
+	 */
+	if (
+		expect.getState().currentTestName !==
+		'Template Part: Template part placeholder: Should insert template part when preview is selected'
```

Comment:
Need to figure out the exact text to use here

Reply:
What's special about this test so that it fails?